### PR TITLE
popup: improve JAWS readability

### LINKED ISSUE #1510: popup: improve JAWS readability
JAWS should read
- [x] `popup_visible` with focusable controls inside as "dialog"
- [x] `content` field

Check and fix divergencies.

### PR DESCRIPTION
**Pull request by sipayRT**
Closes #1510

### CODE REVIEW

**Comment by sipayRT** [comment]
@aristov need your review

File: common.blocks/popup/popup.bemhtml [hunk 1]
```diff
@@ -8,5 +8,11 @@ block('popup')(
             directions : ctx.directions,
             zIndexGroupLevel : ctx.zIndexGroupLevel
         };
+    }),
+    attrs()(function() {
+        return {
+            role : 'dialog',
+            'aria-hidden' : !this.mods.visible
```
Comment (aristov) [inline common.blocks/popup/popup.bemhtml]: Add `aria-hidden` attribute only if it's value is `true` and remove it if `false`.

File: common.blocks/popup/popup.bh.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+            .js({
+                mainOffset : json.mainOffset,
+                secondaryOffset : json.secondaryOffset,
+                viewportOffset : json.viewportOffset,
+                directions : json.directions,
+                zIndexGroupLevel : json.zIndexGroupLevel
+            })
+            .attrs({
+                role : 'dialog',
+                'aria-hidden' : !ctx.mod('visible')
```
Comment (aristov) [inline common.blocks/popup/popup.bh.js]: The same as above: omit `aria-hidden="false"`.

File: common.blocks/popup/popup.js [hunk 3]
```diff
@@ -45,14 +45,16 @@ provide(BEMDOM.decl(this.name, /** @lends popup.prototype */{
                 this
                     ._captureZIndex()
                     ._bindToParentPopup()
-                    .bindTo('pointerpress pointerclick', this._setPreventHideByClick);
+                    .bindTo('pointerpress pointerclick', this._setPreventHideByClick)
+                    .domElem.removeAttr('aria-hidden');
```
Comment (aristov) [inline common.blocks/popup/popup.js:49]: Good =)

**Comment by sipayRT** [comment]
@aristov updated
Added tmpl-spec for popup in visible state

**Comment by aristov** [comment]
LGTM

**Pull request merged**